### PR TITLE
Strict types for chrome.contextMenus.*

### PR DESCRIPTION
**Pull request by bershanskiy**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:
If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://developer.chrome.com/docs/extensions/reference/contextMenus/>

### CODE REVIEW

**Comment by bershanskiy** [comment]
This patch adds tests for `chrome.contextMenus.*` APIs and replaces few `any` types with stricter types (supported by the linked documentation).

File: types/chrome/index.d.ts [hunk 1]
```diff
@@ -1538,7 +1538,7 @@ declare namespace chrome.contextMenus {
          * Since Chrome 35.
          * The parent ID, if any, for the item clicked.
          */
-        parentMenuItemId?: any;
+        parentMenuItemId?: number | string | undefined;
```
Comment (nicolas377) [inline types/chrome/index.d.ts:1541]: you don't have to include undefined, since that's automatically added for optional values

**Review by nicolas377 — CHANGES_REQUESTED** [review]
undefined doesn't need to be a type for optional types, since its automatically added

**Comment by bershanskiy** [comment]
> undefined doesn't need to be a type for optional types, since its automatically added

Since all other optional elements have explicit `undefined`, I thought  was some kind of style convention. I removed it now.

**Review by nicolas377 — APPROVED** [review]
LGTM!

**Comment by bershanskiy** [comment]
Ready to merge

**Pull request merged**